scripts/manage_flavours.py

```diff
@@ -57,6 +64,52 @@ async def update_flavour(flavour_data: dict, logger: logging.Logger):
             logger.info("Flavor %s update is completed", flavour_data["name"])
 
 
+async def prune_flavours(flavours_data: [], logger: logging.Logger):
+    async with database.Session() as db:
+        flavour_names_in_config = list(pluck("name", flavours_data))
+        flavours_in_db = await pf_crud.list_flavours(db)
+
+        logger.info(
+            "Found %d flavours in database, %d flavours in config",
+            len(flavours_in_db),
+            len(flavours_data),
+        )
+
+        orphaned_flavours = []
+        for flavour in flavours_in_db:
+            if flavour.name not in flavour_names_in_config:
+                orphaned_flavours.append(flavour.id)
+                logger.info("    %s (orphaned)", flavour.name)
+            else:
+                logger.info("    %s", flavour.name)
+
+        logger.info("")
+
+        if not orphaned_flavours:
+            logger.info("There's no orphaned flavours, exitting.")
+            return
+
+        confirmation = (
+            input(
+                "Are you sure want to delete orphaned flavours?\n"
+                "This operation cannot be undone!! (yes/no): "
+            )
+            == 'yes'
+        )
```

Comment:
I think that it's unnecessary to add additional confirmation

Reply:
I added it because there's no undo but I'm fine with removing it if you think unnecessary.

Reply:
I added an option to skip confirmation instead as @zklevsha suggested.